cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -88,14 +93,21 @@
   private final SSHPublicKey publicKey;
   private final Map<String, String> dataprocProperties;
 
+  private final String clustermetadata;
+  private final String networktags;
+  private final String initactions;
+  private final boolean enableautoscaling;
+  private final String autoscalingpolicy;
```

Comment:
Why need both enableAutoScaling and autoScalingPolicy?? We can just have the `autoScalingPolicy`. If it is null, then there is no auto-scaling.

Reply:
This was implemented as per Spec from Bhooshan. Need to check with him if we have to remove it.